packages/block-library/src/post-categories/block.json

```diff
@@ -2,6 +2,9 @@
 	"name": "core/post-categories",
 	"category": "design",
 	"attributes": {
+		"term": {
+			"type": "object"
```

Comment:
Why the "object"? why not just the term slug, it will make it easier to create variations?

Reply:
Right, it makes sense. I've moved the entity fetching around a lot and I ended up being more interested in how to create dynamic variations than their attributes. 😄 

As far as I could see:
- `getEntityRecord` uses the `slug` (e.g. "category")
- `useEntityProp` uses the `rest_base` (e.g. "categories")

With the taxonomy list handy, we don't need to have both properties stored. I'll update this.